Kitodo/src/main/webapp/WEB-INF/web.xml

```diff
@@ -88,6 +88,12 @@
         <!--<param-value>none</param-value>-->
     <!--</context-param>-->
 
+    <listener>
```

Comment:
As the application build as a servlet 3.0+ application, then this is not needed any more, according to the [readme of the project](https://github.com/mjiderhamn/classloader-leak-prevention). Maybe you must adjust the pom entry too.